src/chrome/content/rules/Desjardins.xml

```diff
@@ -1,9 +1,14 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://accesd.desjardins.com/ => https://accesd.desjardins.com/: (28, 'Connection timed out after 10001 milliseconds')
+Fetch error: http://accesd.affaires.desjardins.com/ => https://accesd.affaires.desjardins.com/: (28, 'Connection timed out after 10000 milliseconds')
+
```

Comment:
> I think this is correct to disable because there is no HTTP version.

Would it be correct to infer from this that https-everywhere's threat model only includes passive observers? I had assumed up until now that the threat model included a mitm that could change traffic in transit, rather than a simple passive dragnet.

I'm guessing that torbrowser does include an active adversary (@ioerror would this guess be correct?), so perhaps there ought to be a platform="tor" for coverage beneficial to only an active mitm scenario.

Further, perhaps the current threat model assumes that our users are as clued up as we are, in that you and I realize plaintext content is to be considered as rendering bug fodder (and never to be trusted). However, as Ian Goldberg illustrated in last year's shmoocon keynote, average users really aren't that clued up. Presented with a plaintext copy and a padlock favicon, hilarity will ensue.

Clued up users would suffer more distinguishable behavior in their traffic, but I think that's a small cost in comparison.

I really think you should consider adding an adversary that can copy and inject web pages into the https-everywhere threat model.